test/type_hint_tests/size.py

```diff
@@ -0,0 +1,9 @@
+import torch
+input = []
+input.append(torch.tensor([1.0, 2.0, 3.0, 4.0]))
+input.append(torch.tensor([[1.0, 2.0, 3.0, 4.0]]))
+input.append(torch.tensor([[[1.0, 2.0, 3.0, 4.0]]]))
+input[0].shape[0]
+input[1].shape[1]
+input[2].shape[2]
+
```

Comment:
I think flake8 is complaining about this line -- we should delete it

Reply:
Do I remove the endline or add it, in `test_type_hint`, you asked me to first remove it, then add it. Here, it's required.